service/history/replication/executable_backfill_history_events_task.go

```diff
@@ -203,43 +203,6 @@ func (e *ExecutableBackfillHistoryEventsTask) HandleErr(err error) error {
 	}
 }
 
-func (e *ExecutableBackfillHistoryEventsTask) MarkPoisonPill() error {
```

Comment:
This moved to ExecutableTask, so we don't need to copy logic into every task.

Reply:
nice!